src/main/java/frc/robot/AutoFactory.java

```diff
@@ -145,7 +175,104 @@ public Supplier<Command> getPathFromWaypoints(Rotation2d goalEndRotationHolonomi
         return pathCommand;
     }
 
-    public enum CharacterizationRoutine {
+    /**
+     * Which coral station we are using
+     */
+    public static enum CoralStation {
+        /**
+         * The left-hand coral station
+         */
+        LEFT,
+        /**
+         * The right-hand-coral station
+         */
+        RIGHT
+    }
+
+    public static enum StartingPosition {
```

Comment:
pls add javadocs here for clarity on your starting letter code meanings. i get it but again just for clarity pls.